### ISSUE TITLE
the mean and std of smallNORB dataset

### ISSUE THREAD

**Issue by HJLYU0519** [www0wwwjs1/Matrix-Capsules-EM-Tensorflow#49]
I want to realize the first normalization of smallNORB, but I could not get the value of mean and variance(std) in  data/smallNORB.py     
           177 line:  mean, variance = tf.nn.moments(image, [0, 1, 2])
I am sure both mean and variance is scalar.
 is there some way to get the mean and variance(std) of smallNORB dataset?

Thanks so much if you could help me!

**Comment by HJLYU0519** [comment]
if I print(mean, variance), I could just get some attributes but not the value of mean and std
